terraform/terraform_environment/ecs_api_cron.tf

```diff
@@ -0,0 +1,82 @@
+
+//-------------------------
+// Permissions
+
+// What services can assume the role
+data "aws_iam_policy_document" "cloudwatch_events_assume_policy" {
+  statement {
+    effect  = "Allow"
+    actions = ["sts:AssumeRole"]
+    principals {
+      identifiers = ["events.amazonaws.com"]
+      type        = "Service"
+    }
+  }
+}
+
+resource "aws_iam_role" "cloudwatch_events_ecs_role" {
+  name               = "${local.environment}-cloudwatch_events_ecs_cron"
+  assume_role_policy = data.aws_iam_policy_document.cloudwatch_events_assume_policy.json
+}
+
+//---
+
+data "aws_iam_policy_document" "cloudwatch_events_role_policy" {
+  statement {
+    effect    = "Allow"
+    actions   = ["ecs:RunTask"]
+    resources = ["*"]
```

Comment:
I think we can limit this to a specific cluster, which would be a good limitation on the policy
`aws_ecs_cluster.online-lpa.arn` should do it.